Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.17.6-bullseye
```

Comment:
1. I would suggest using an alpine-based image instead (with git on top of it). The bullseye image takes up almost 1GB in comparison with 329Mb of an alpine one.
2. Let's use a multistage docker build. After a Goxygen binary is built, we don't need its source code in the image as well as the Go compiler, and executor. You can copy the `/app/goxygen` to the new image (an alpine-based one). In the end, the image would be only 24Mb in size.